Routers/Routers/Routers/DisconnectedGraphException.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Routers;
+
+/// <summary>
+/// A class for creating custom exceptions
+/// </summary>
+public class DisconnectedGraph : Exception
```

Comment:
```suggestion
public class DisconnectedGraphException : Exception
```
Все исключения в .NET именуются с суффиксом Exception